gpytorch/mlls/predictive_log_likelihood.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+from ._approximate_mll import _ApproximateMarginalLogLikelihood
+
+
+class PredictiveLogLikelihood(_ApproximateMarginalLogLikelihood):
+    r"""
+    An alternative objective function for approximate GPs, proposed in `Jankowiak et al., 2019`_.
+    It typically produces predictive variances than the :obj:`gpytorch.mlls.VariationalELBO` objective.
+
+    .. math::
+
+       \begin{align*}
+          \mathcal{L}_\text{ELBO} &=
+          \mathbb{E}_{p_\text{data}( y, \mathbf x )} \left[
+            \log p( y \! \mid \! \mathbf x)
+          \right] - \beta \: \text{KL} \left[ q( \mathbf u) \Vert p( \mathbf u) \right]
+          \\
+          &\approx \sum_{i=1}^N \log \mathbb{E}_{q(\mathbf u)} \left[
+            \int p( y_i \! \mid \! f_i) p(f_i \! \mid \! \mathbf u) \: d f_i
+          \right] - \beta \: \text{KL} \left[ q( \mathbf u) \Vert p( \mathbf u) \right]
+       \end{align*}
+
+    where :math:`N` is the amount of data, :math:`q(\mathbf u)` is the variational distribution for
+    the inducing function values, and `p(\mathbf u)` is the prior distribution for the inducing function
+    values.
+
+    :math:`\beta` is a scaling constant that reduces the regularization effect of the KL
+    divergence. Setting :math:`\beta=1` (default) results in the true variational ELBO.
```

Comment:
not quite right "Setting :math:`\beta=1` (default) results in the true variational ELBO"